### PR TITLE
CUMULUS-1363: use npm instead of yarn

### PR DESCRIPTION
**Pull request by jennyhliu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -96,7 +97,7 @@
     "gulp-size": "^2.1.0",
     "gulp-sourcemaps": "^1.6.0",
     "gulp-uglify-es": "^1.0.4",
-    "gulp-useref": "3.1.5",
+    "gulp-useref": "^4.0.0",
```
Comment (markdboyd) [inline package.json]: I had some reservations about updating to this version, because it seems like `gulp-useref` is still relying on versions of `event-stream` that may be malicious:

https://github.com/jonkemp/gulp-useref/issues/256

Supposedly it's fine, but it still seemed disconcerting to me

Comment (jennyhliu) [reply package.json]: The latest `event-stream 4.0.1` no longer has malicious dependency `flatmap-stream` and should be safe, and `npm audit` doesn't report any errors.

Comment (markdboyd) [reply package.json]: OK, sounds good

File: .circleci/config.yml [hunk 2]
```diff
@@ -6,38 +6,23 @@ jobs:
     steps:
       - checkout
```
Comment (flamingbear) [inline .circleci/config.yml]: Why did you remove the cache step?

Comment (flamingbear) [reply .circleci/config.yml]: I'm guessing because we went to use `npm ci` which nukes the node_modules directory before anything...

File: app/src/js/components/Executions/execution-status-graph.js [hunk 3]
```diff
@@ -2,7 +2,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import dagre from 'dagre-d3v4';
+import dagre from 'dagre-d3';
 import * as d3 from 'd3';
 
 // dagre-d3v4 expects d3 to be attached to the window
```
Comment (flamingbear) [inline app/src/js/components/Executions/execution-status-graph.js]: We're not using dagre-d3v4 anymore.  Need to update comment.

File: .circleci/config.yml (hunk 2 above)
Comment (flamingbear) [reply .circleci/config.yml]: Is this much slower than before?  It seems ok to me, just curious.

Comment (jennyhliu) [reply .circleci/config.yml]: > I'm guessing because we went to use `npm ci` which nukes the node_modules directory before anything...

Yes.  If the dependencies in `package-lock.json` don't match with `package.json`, `npm ci` will exit with error.
I guess it would be slower, but acceptable.
